app/controllers/catalog_controller.rb

```diff
@@ -329,6 +329,8 @@ class CatalogController < ApplicationController # rubocop:disable Metrics/ClassL
     }, if: lambda { |_context, _field_config, document|
       document.containers.present?
     }
+    config.add_component_field 'date', field: 'normalized_date_ssm'
```

Comment:
```suggestion
    config.add_component_field 'date', accessor: 'normalized_date'
```

Do we want to use the model accessor for this? 🤷‍♂️

Reply:
The model accessor doesn't do anything now except return the field, so I don't know why that would be better. It doesn't matter to me one way or the other.

Reply:
Ok. I wasn't sure from https://github.com/projectblacklight/arclight/issues/1336 if you intended/expected to remove that solr field at some point.